### PR TITLE
*sum: add Tamil translation

### PR DESCRIPTION
**Pull request by arunisaac**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.ta/common/b2sum.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# b2sum
+
+> BLAKE2 மறையீட்டு சரிகாண்தொகையைக் கணி.
+> மேலும் விவரத்திற்கு: <https://www.gnu.org/software/coreutils/b2sum>.
```
Comment (navarroaxel) [inline pages.ta/common/b2sum.md:4]: We use this format:
```suggestion
> மேலும் தகவல்: <https://www.gnu.org/software/coreutils/b2sum>.
```

File: pages.ta/common/cksum.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# cksum
+
+> கோப்பின் CRC சரிகாண்தொகையைக் கணித்து அதில் எத்தனை எண்ணிருமிகளுள்ளன என்றெண்ணு.
+> குறிப்பு: பழைய Unix கணினிகளில் CRC கணிமுறை மாறலாம்.
+> மேலும் விவரத்திற்கு: <https://www.gnu.org/software/coreutils/cksum>.
```
Comment (navarroaxel) [inline pages.ta/common/cksum.md:5]:
```suggestion
> மேலும் தகவல்: <https://www.gnu.org/software/coreutils/cksum>.
```

File: pages.ta/common/md5sum.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# md5sum
+
+> MD5 மறையீட்டு சரிகாண்தொகையைக் கணி.
+> மேலும் விவரத்திற்கு: <https://www.gnu.org/software/coreutils/md5sum>.
```
Comment (navarroaxel) [inline pages.ta/common/md5sum.md:4]:
```suggestion
> மேலும் தகவல்: <https://www.gnu.org/software/coreutils/md5sum>.
```

File: pages.ta/common/sha1sum.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# sha1sum
+
+> SHA1 மறையீட்டு சரிகாண்தொகையைக் கணி.
+> மேலும் விவரத்திற்கு: <https://www.gnu.org/software/coreutils/sha1sum>.
```
Comment (navarroaxel) [inline pages.ta/common/sha1sum.md:4]:
```suggestion
> மேலும் தகவல்: <https://www.gnu.org/software/coreutils/sha1sum>.
```

File: pages.ta/common/sha224sum.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# sha224sum
+
+> SHA224 மறையீட்டு சரிகாண்தொகையைக் கணி.
+> மேலும் விவரத்திற்கு: <https://www.gnu.org/software/coreutils/manual/html_node/sha2-utilities.html>.
```
Comment (navarroaxel) [inline pages.ta/common/sha224sum.md:4]:
```suggestion
> மேலும் தகவல்: <https://www.gnu.org/software/coreutils/manual/html_node/sha2-utilities.html>.
```

File: pages.ta/common/sha256sum.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# sha256sum
+
+> SHA256 மறையீட்டு சரிகாண்தொகையைக் கணி.
+> மேலும் விவரத்திற்கு: <https://www.gnu.org/software/coreutils/manual/html_node/sha2-utilities.html>.
```
Comment (navarroaxel) [inline pages.ta/common/sha256sum.md:4]:
```suggestion
> மேலும் தகவல்: <https://www.gnu.org/software/coreutils/manual/html_node/sha2-utilities.html>.
```

File: pages.ta/common/sha384sum.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# sha384sum
+
+> SHA384 மறையீட்டு சரிகாண்தொகையைக் கணி.
+> மேலும் விவரத்திற்கு: <https://www.gnu.org/software/coreutils/manual/html_node/sha2-utilities.html>.
```
Comment (navarroaxel) [inline pages.ta/common/sha384sum.md:4]:
```suggestion
> மேலும் தகவல்: <https://www.gnu.org/software/coreutils/manual/html_node/sha2-utilities.html>.
```

File: pages.ta/common/sha512sum.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# sha512sum
+
+> SHA512 மறையீட்டு சரிகாண்தொகையைக் கணி.
+> மேலும் விவரத்திற்கு: <https://www.gnu.org/software/coreutils/manual/html_node/sha2-utilities.html>.
```
Comment (navarroaxel) [inline pages.ta/common/sha512sum.md:4]:
```suggestion
> மேலும் தகவல்: <https://www.gnu.org/software/coreutils/manual/html_node/sha2-utilities.html>.
```

File: pages.ta/common/sum.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# sum
+
+> கோப்பின் சரிகாண்தொகையைக் கணித்து அதில் எத்தனைத் தொகுதிகளுள்ளன என்றெண்ணு.
+> இதுத் தற்காலத்திய `cksum` கட்டளைக்கு முன் தோன்றியது.
+> மேலும் விவரத்திற்கு: <https://www.gnu.org/software/coreutils/sum>.
```
Comment (navarroaxel) [inline pages.ta/common/sum.md:5]:
```suggestion
> மேலும் தகவல்: <https://www.gnu.org/software/coreutils/sum>.
```

File: pages.ta/common/cksum.md [hunk 10]
```diff
@@ -0,0 +1,9 @@
...
+
+- கோப்பின் 32-இருமி சரிகாண்தொகையையும் எண்ணிருமி அலகில் கோப்பளவையும் பெயரையும் காட்டு:
+
+`cksum {{கோப்பு}}`
```
Comment (navarroaxel) [inline pages.ta/common/cksum.md:9]: Why you didn't translate the `path/to/file` placeholder as-is?

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by arunisaac** [comment]
> +> மேலும் விவரத்திற்கு: <https://www.gnu.org/software/coreutils/b2sum>.
>
> ```suggestion
>> மேலும் தகவல்: <https://www.gnu.org/software/coreutils/b2sum>.
> ```

"மேலும் விவரத்திற்கு" is a better translation. "மேலும் தகவல்" is even
slightly wrong grammatically. If there are any pages that have "மேலும்
தகவல்", I'll be happy to create a pull request fixing those pages.

> +`cksum {{கோப்பு}}`
>
> Why you didn't translate the `path/to/file` placeholder as-is?

Tamil is an agglutinative language. This means that what are phrases in
English often get translated to one giant word in Tamil. These giant
words can be quite a mouthful and hard to read. For example,
"path/to/file" translates to "கோப்புப்பாதை", an agglutination of two
words. I feel that just "கோப்பு" (meaning file) sounds clearer and more
concise.

**Comment by navarroaxel** [comment]
@arunisaac Could you create a PR to fix this? https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py

**Comment by arunisaac** [comment]
Done!

**Comment by navarroaxel** [comment]
@arunisaac awesome, then only 1 suggestion is pending.
Also, you can add the Tamil common arguments here https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md and https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md

**Comment by arunisaac** [comment]
Done!

File: pages.ta/common/cksum.md (hunk 10 above)
Comment (marchersimon) [reply pages.ta/common/cksum.md:9]: Answered below:

> Tamil is an agglutinative language. This means that what are phrases in
English often get translated to one giant word in Tamil. These giant
words can be quite a mouthful and hard to read. For example,
"path/to/file" translates to "கோப்புப்பாதை", an agglutination of two
words. I feel that just "கோப்பு" (meaning file) sounds clearer and more
concise.

**Review by marchersimon — APPROVED** [review]
Thanks @arunisaac

**Review by navarroaxel — APPROVED** [review]

**Comment by arunisaac** [comment]
Thank you!

**Pull request merged**